tools/normalize_mri_protocol_range_data.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * This script is the part of the second step for normalizing the mri_protocol table.
+ * It populates the new min & max columns for each field that can contain ranges and
+ * returns SQL statements to remove the old %_range columns.
+ *
+ * "Usage: php normalize_mri_protocol_range_data.php [tosql]";
+ * "Example: php normalize_mri_protocol_range_data.php tosql";
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Liza Levitis <llevitis.mcin@gmail.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris
+ */
+require_once 'generic_includes.php';
+require_once 'Database.class.inc';
+require_once 'Utility.class.inc';
+
+// set default arguments
+$printToSQL = false;
+// SQL output
+$output     = "";
+
+// check if the 'tosql' option is passed
+if (!empty($argv[1]) && $argv[1] == "tosql") {
```

Comment:
I don't find "tosql" very explicit in terms of option. Maybe rename it to something more intuitive?

Reply:
@cmadjar it is the option used by other script, so I would keep it as is for consistancy

Reply:
Ah, ok. Did not know that. Let's keep it to tosql then. Thanks @PapillonMcGill for the clarification.